fxa/tests/test_core.py

```diff
@@ -176,6 +176,32 @@ def wait_for_email(m):
                                                  m["headers"]["x-verify-code"])
         self.assertEquals(response, {})
 
+    def test_send_unblock_code(self):
+        acct = TestEmailAccount()
+        session1 = self.client.create_account(
+            email=acct.email,
+            stretchpwd=DUMMY_STRETCHED_PASSWORD,
+        )
+        self._accounts_to_delete.append(acct)
+
+        # Initiate sending unblock code
+        response = self.client.send_unblock_code(acct.email)
+        self.assertEquals(response, {})
+
+        m = acct.wait_for_email(lambda m: "x-unblock-code" in m["headers"])
+        if not m:
+            raise RuntimeError("Unblock code email was not received")
+
+        code = m["headers"]["x-unblock-code"]
+        self.assertTrue(len(code) > 0)
+
+        # XXX TODO: the live server doesn't block the account, so we can't
+        # really test login flow with an unblock code
```

Comment:
Actually, since this talks to the dev server by default, I believe it will block login attempts for addresses that match `/^(block.*@restmail\.net)$/`.  If you create the TestEmailAccount to match that pattern then the I think we could get a working test here.